app/queries/get_reasons_for_rejection_from_application_choices.rb

```diff
@@ -0,0 +1,17 @@
+class GetReasonsForRejectionFromApplicationChoices
```

Comment:
I still think it's worth adding the method to run the query in this class rather than the controller. Also it would be worth using SQL params rather than string interpolation for the date range, if only because it would avoid Brakeman complaining about possible SQL injection.

Reply:
We managed to extract the db query into the the Query object and get rid of interpolation 👍